### DISCUSSION TITLE
Improve the styling of asset thumbnails

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#13328] [Ideas]
I often find that the current styling of asset thumbnails is not well suited to all types of images. 

Specifically, if an image has an irregular shape - like an illustration with surrounding white space - it is not visually "anchored" (by alignment) to the filename that appears bellow, and this problem is compounded in asset indexes, creating a somewhat chaotic impression:

<img width="500" alt="image" src="https://github.com/craftcms/cms/assets/19857599/ea0d5acf-7926-4a26-ab5a-227e3b5bb268">

I would think that thumbnails would be better displayed in a kind of "frame", like a card, that clearly defines the image area, creates a visual grid and helps with alignment and grouping:

<img width="500" alt="image" src="https://github.com/craftcms/cms/assets/19857599/0f9855bb-f2dc-49a8-9998-f5aeefd99c0e">

In the above example I have opted to also hide the filename and show it only on hover (see the first thumb in the top row), but this is not necessary or desirable for all kinds of projects (would be nice to have it as an option per volume or per field maybe?).

I have for some time now tested this customized look - with the help of the `Control Panel CSS` plugin - in a number of projects and I think it just works better, in asset indexes (large thumbnail view) and in the entry edit screen, in asset fields (large thumbnail view):

Without card styling
<img width="438" alt="image" src="https://github.com/craftcms/cms/assets/19857599/5d79f74d-a626-4d3a-a379-d0ccded9aed3">

With card styling
<img width="439" alt="image" src="https://github.com/craftcms/cms/assets/19857599/1b4b4456-4faf-4a6d-955e-23b92f3b8667">

Anyway, that's my 2 cents! 

Thanks for reading, and as always I would love to hear opinions :)

**Comment by alexanderloewe** [comment]
Would like something like this. But I think the light blue background should be lighter.

**Reply by thupsi** [reply]
Thanks! Sure, I have used a value that already exists in the CP, but in reality it just has to have enough contrast against white.
